control_plane/src/compute.rs

```diff
@@ -74,22 +74,29 @@ impl ComputeControlPlane {
     }
 
     // FIXME: see also parse_point_in_time in branches.rs.
-    fn parse_point_in_time(
+    fn parse_timeline_spec(
```

Comment:
This deserves a comment, explaining what 'replica_of' means. Is it valid to use "foo@0/12345" syntax to specify a specific LSN, and *also* have 'replica_of'?